### PR TITLE
feat(timeline): add capability to change the layout of button

### PR DESCRIPTION
**Pull request by stonebuzz**
Add user pref option to split or not timeline action button.

Standard layout (```merged```)
image

New layout (```splitted```)
image

NB : do not hide buttons when form (```task```, ```followup```, ```solution``` ... ) are open

| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

**Comment by stonebuzz** [comment]
image

**Comment by AdrienClairembault** [comment]
I think you should try to flex the container : 

image

Without this I get a 2 row display despite having enough space on my screen to fit all buttons:

image

**Comment by AdrienClairembault** [comment]
Also, an open question: maybe we could gain space by shortening the labels ?
The first button is very short "Réponse" but the others are more verbose like "Ajouter une solution" instead of just "Solution".
I'm afraid that there wont be enough space on the screen for some languages.

**Review by AdrienClairembault — CHANGES_REQUESTED** [review]
See comments.

**Comment by AdrienClairembault** [comment]
I think validation was a different color in GLPI 9.5 (if you want to keep it consistent).
Some kind of lime green if I recall correctly ?

**Review by AdrienClairembault — APPROVED** [review]

File: templates/components/itilobject/timeline/filter_timeline.html.twig [hunk 1]
```diff
@@ -81,7 +81,7 @@
    }) %}
 {% endif %}
 
-<div class="filter-timeline float-end position-relative">
+<div class="filter-timeline float-end position-relative ms-auto">
```
Comment (orthagh) [inline templates/components/itilobject/timeline/filter_timeline.html.twig:84]: i guess the float-end class is not needed anymore

Comment (stonebuzz) [reply templates/components/itilobject/timeline/filter_timeline.html.twig:84]: ```ms-auto``` is always needed

Comment (AdrienClairembault) [reply templates/components/itilobject/timeline/filter_timeline.html.twig:84]: yes but not `float-end` 😉

Comment (stonebuzz) [reply templates/components/itilobject/timeline/filter_timeline.html.twig]: Oo, sorry !

**Review by orthagh — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**